lib/praxis/docs/open_api/schema_object.rb

```diff
@@ -57,9 +63,15 @@ def dump_schema(shallow: false, allow_ref: false)
 
           # Tag on OpenAPI specific requirements that aren't already added in the underlying JSON schema model
           # Nullable: (it seems we need to ensure there is a null option to the enum, if there is one)
-          if @attribute_options[:null]
-            h[:nullable] = @attribute_options[:null]
-            h[:enum] = h[:enum] + [nil] if h[:enum] && !h[:enum].include?(nil)
+          if default_nullability
+            unless @attribute_options[:required]
```

Comment:
I'm a little unsure about this...requirement and nullability are completely orthogonal...so seeing a dependent condition for both smells wrong...

Reply:
Fair point; I was cutting corners and noticed that in our codebase we only ever annotate with EITHER requirement or nullability but that kind of thinking doesn't scale particularly well to defining specs. I'll take a step back and try to mark requirement distinctly from nullability instead.